TFG-Labs/custom-shipping-step-by-items
# fix submitting custom delivery fields

### What problem is this solving?
- submitting extra delivery fields 

#### How it works

#### How to test it?
- fill in all the required fields with a mixed cart
- then go to payment, should update the custom order data correctly.

[Workspace](https://delveryshipping--thefoschini.myvtex.com/checkout#/payment)

### Screenshots/Video example usage:
- Desktop
<img width="572" alt="Screenshot 2022-12-22 at 09 37 40" src="https://user-images.githubusercontent.com/8531068/209082078-93533e00-23b9-465b-9bdf-b9ef13c99796.png">


### Related to / Depends on

#### How does this PR make you feel? :link:
![]()
